app.js

```diff
@@ -1,75 +1,165 @@
 const author = document.querySelector('#author');
 const displayArea = document.querySelector('#showBooks');
+const list = document.querySelector('#booksshow');
+const addNew = document.querySelector('#new');
+const contact = document.querySelector('#contact');
+const addBook = document.querySelector('#adding');
+const contactInfo = document.querySelector('#reachme');
+const dates = document.querySelector('#date');
 
-// Display the books stpr
-function displayBooks() {
-  const wrapper = document.createElement('div');
-  const bookShelfstr = localStorage.getItem('books');
-  const bookArray = JSON.parse(bookShelfstr);
-  bookArray.forEach((element, index) => {
-    const displayTitle = document.createElement('p');
-    const displayAuth = document.createElement('p');
-    const deletebtn = document.createElement('div');
-    const container = document.createElement('div');
-    const line = document.createElement('hr');
-    displayTitle.innerText = element.title;
-    displayAuth.innerText = element.author;
-    deletebtn.innerHTML = `<button onclick='removeBook(${index})'>Remove</button>`;
-    deletebtn.classList.add('deleteBook');
-    container.appendChild(displayTitle);
-    container.appendChild(displayAuth);
-    container.appendChild(deletebtn);
-    container.appendChild(line);
-    wrapper.appendChild(container);
-  });
-  displayArea.appendChild(wrapper);
-}
+class Books {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 
-/* Take the data store it  */
-const storeObj = {};
+  storeData() {
+    const bookTitle = title.value;
+    const bookAuthor = author.value;
+    const book = new Books(bookTitle, bookAuthor);
+    // Check if local storage is empty
+    if (localStorage.getItem('books') === null) {
+      const bookShelf = [];
+      bookShelf.push(book);
+      localStorage.setItem('books', JSON.stringify(bookShelf));
+    } else {
+      const bookShelfstr = localStorage.getItem('books');
+      const bookArray = JSON.parse(bookShelfstr);
+      bookArray.push(book);
+      localStorage.setItem('books', JSON.stringify(bookArray));
+    }
+    // Clear input
+    title.value = '';
+    author.value = '';
+    this.displayBooks();
+  }
 
-// Function to store in local storage
-function storeData() {
-  // Check if local storage is empty
-  if (localStorage.getItem('books') === null) {
-    const bookShelf = [];
-    bookShelf.push(storeObj);
-    localStorage.setItem('books', JSON.stringify(bookShelf));
-  } else {
+  displayBooks() {
+    const wrapper = document.createElement('div');
+    const line = document.createElement('hr');
     const bookShelfstr = localStorage.getItem('books');
+    const tits = document.createElement('div');
+    tits.innerText = this.author;
     const bookArray = JSON.parse(bookShelfstr);
-    bookArray.push(storeObj);
+    bookArray.forEach((element, index) => {
+      const displayTitle = document.createElement('p');
```

Comment:
- [ ] At the app start if you don't have anything stored in the localStorage the `bookArray.forEach` raise a null exception. To avoid that you can use the JS optional chaining which prevents errors when the object is null or undefined.
`bookArray?.forEach`

<img width="415" alt="Screenshot 2022-11-30 at 2 39 04 PM" src="https://user-images.githubusercontent.com/40503704/204810990-5d5f9ad1-afb8-44f9-b9dc-932b58f2e877.png">